src/components/inputs/Slider/Slider.tsx

```diff
@@ -111,10 +118,21 @@ export function Slider({
     dispatch({ type: 'SET_TRACK_RECT', payload: { left, width } });
   }, [dragging]);
 
+  useEffect(() => {
+    if (!isFirstRender.current) {
+      dispatchChangeEvent(onChange);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [onChange, values]);
+
+  useEffect(() => {
+    isFirstRender.current = false;
+  }, []);
+
```

Comment:
Is there a reason why you chose not to put this in the same useEffect? ie:

```
  useEffect(() => {
    if (!isFirstRender.current) {
      dispatchChangeEvent(onChange);
    } else {
     isFirstRender.current = false;
    }
  }, [onChange, values]);

```

Reply:
No, you're right thanks!
Changed it :)